src/include/platform/internal/GenericPlatformManagerImpl_Zephyr.ipp

```diff
@@ -180,6 +195,20 @@ void GenericPlatformManagerImpl_Zephyr<ImplClass>::_RunEventLoop(void)
     }
     mShouldRunEventLoop = true;
 
+#if CHIP_SYSTEM_CONFIG_USE_SOCKETS
```

Comment:
These if/else/endif are scattered throughout the code.

Would it be reasonably clean to create a separte IPP file one for ZephyrSockets and one ZephyrNoSockets ? That way we have one if for includes and don't split out the code as much.